test/protocol-buffer-conversion-test.ts

```diff
@@ -555,4 +564,286 @@ describe('Protocol Buffer Conversion', function(): void {
     assert.deepEqual(signer?.signingPublicKey, signingPublicKey)
     assert.deepEqual(signer?.transactionSignature, transactionSignature)
   })
+
+  // Transaction
+  it('Convert PAYMENT Transaction, all common fields set', function(): void {
+    // GIVEN a Transaction protocol buffer with all common fields set.
+    const account = 'r123'
+    const fee = '1'
+    const sequence = 2
+    const signingPublicKey = new Uint8Array([1, 2, 3])
+    const transactionSignature = new Uint8Array([4, 5, 6])
+    const accountTransactionID = new Uint8Array([7, 8, 9])
+    const rawValue = 4
+    const flags = RippledFlags.checkFlag(
+      rawValue,
+      RippledFlags.TF_PARTIAL_PAYMENT,
+    )
+    const lastLedgerSequence = 5
+    const memoData = new Uint8Array([1, 2, 3])
+    const memoFormat = new Uint8Array([4, 5, 6])
+    const memoType = new Uint8Array([7, 8, 9])
+
+    // memo proto
+    const memoDataProto = new MemoData()
+    memoDataProto.setValue(memoData)
+    const memoFormatProto = new MemoFormat()
+    memoFormatProto.setValue(memoFormat)
+    const memoTypeProto = new MemoType()
+    memoTypeProto.setValue(memoType)
+
+    const memoProto = new Memo()
+    memoProto.setMemoData(memoDataProto)
+    memoProto.setMemoFormat(memoFormatProto)
+    memoProto.setMemoType(memoTypeProto)
+
+    // signer proto
+    const signerAccountAddressProto = new AccountAddress()
+    signerAccountAddressProto.setAddress(account)
+    const signerAccountProto = new Account()
+    signerAccountProto.setValue(signerAccountAddressProto)
+
+    const signerSigningPublicKeyProto = new SigningPublicKey()
+    signerSigningPublicKeyProto.setValue(signingPublicKey)
+
+    const signerTransactionSignatureProto = new TransactionSignature()
+    signerTransactionSignatureProto.setValue(transactionSignature)
+    const signerProto = new Signer()
+    signerProto.setAccount(signerAccountProto)
+    signerProto.setSigningPublicKey(signerSigningPublicKeyProto)
+    signerProto.setTransactionSignature(signerTransactionSignatureProto)
+
+    const sourceTag = 6
```

Comment:
I'd move this down to where you build the source tag on line 645

Reply:
moved it up to where all the other real values are defined, for consistency